### ISSUE TITLE
Wpscan

### ISSUE THREAD

**Issue by ghost** [RafaelRfs/R5FS-Phone-Termux#1]
Any plan to add wpscan?

**Comment by RafaelRfs** [comment]
Yes, I'm fixing any of the bugs with script and i think to update and add  it!!!

**Comment by ghost** [comment]
Great man

**Comment by mrbanzeras** [comment]
Usin wpscan in termux with chroot kali linux image  tryed many ways none works in termux:)
